src/enter.js

```diff
@@ -26,6 +26,6 @@ export default class Enter extends Feature {
 		this.listenTo( editingView, 'enter', ( evt, data ) => {
 			editor.execute( 'enter' );
 			data.preventDefault();
-		} );
+		}, 'low' );
```

Comment:
And there is `low` added after this listener (I am not sure now how this chain of dependencies and requirements in plugins will work, but I hope that `Enter` feature will be loaded as one of first ones.